kedro-org/kedro-viz
# Fix the color of the update dot on the settings icon

Signed-off-by: Tynan DeBold <thdebold@gmail.com>

## Description

Fixes #929 

## Development notes

I've removed the dot from the SVG and moved it to be a sibling element. That way, it isn't affected by the opacity of the icon and can be bright all the time.

## QA notes

Check the Gitpod link. The little dot should be bright. 

@GabrielComymQB regarding the icon not appearing crisp, I think this is a Retina display versus non-Retina issue. It looks really crisp on my laptop's screen (and in the screenshot it's not as crisp as what I see):

<img width="132" alt="image" src="https://user-images.githubusercontent.com/16869061/177301151-00ee8f22-d397-432c-a8c1-a843cde01c2b.png">

On my external 27" monitor, less so, though not terrible.

## Checklist



<a href="https://gitpod.io/#https://github.com/kedro-org/kedro-viz/pull/956"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
